packages/Webkul/Bulkupload/src/Database/Migrations/2019_04_23_115227_create_bulkupload_dataflowprofile_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateBulkuploadDataflowprofileTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('bulkupload_dataflowprofile', function (Blueprint $table) {
```

Comment:
Table name should be bulkupload_data_flow_profiles

Reply:
done